src/mixins/I18n.ts

```diff
@@ -62,11 +62,15 @@ export interface I18nMixin {
 	 * The localized messages, along with a `format` method for formatting ICU-formatted templates.
 	 */
 	localizeBundle<T extends Messages>(bundle: Bundle<T>): LocalizedMessages<T>;
+
+	properties: I18nProperties;
 }
 
-export function I18nMixin<T extends Constructor<WidgetBase<I18nProperties>>>(base: T): T & Constructor<I18nMixin> {
+export function I18nMixin<T extends Constructor<WidgetBase<any>>>(base: T): T & Constructor<I18nMixin> {
```

Comment:
With it being `I18nProperties` means that the author of a the widget needs to have extended the `I18nProperties` at design time in order for the mixin to be able to be applied.

So I couldn't decorate mixin behaviour on to a widget unless the author originally had the foresight to extend the appropriate property object (or objects). This is especially sucky for things like the `Container` that assume the properties interface of the widget it is wrapping.

I picked `any` because in the future I would like to actually remove the requirement for consumers to even extend the `WidgetProperties` (and just have then automagically available in all scenarios).

Hopefully makes sense!